config/manifests/bases/apicast-operator.clusterserviceversion.yaml

```diff
@@ -41,10 +41,10 @@ spec:
 
     ### Upgrading your installation
     The APIcast Operator understands how to run and upgrade between a set of APIcast versions.
-    See [the upgrade guide](https://github.com/3scale/apicast-operator) for more information.
+    See [the upgrade guide](https://github.com/3scale/apicast-operator/blob/v0.8.0/doc/operator-user-guide.md#upgrading-APIcast) for more information.
```

Comment:
dead link, like with the image, we need to tag first.

Reply:
Thank you for comment. Yes, I will recheck this llink after tagging, and will confirm.